treasurer/accounting.csv

```diff
@@ -37,5 +37,7 @@ TransactionDate,Income,Expense,Tag,Reason,Notes,ChequeNumber
 2016-10-31,,2.5,BankFee,BMO Maintanence fee,,
 2016-11-21,,125,Misc,CUPE Instructor honorarium,For Jeff,ch010
 2016-11-30,,2.5,BankFee,BMO Maintanence fee,,
+2017-02-06,,13.48,Misc,Supplies for SWC workshop (Feb 3),For Maddy,ch019
```

Comment:
No, it doesn't really matter. It's in the order by date, which is enough. And even order doesn't matter since this is just a database.

Also, this line should have been added in a previous PR, but I forgot to include it.